core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -73,6 +73,8 @@ impl SchedulerController {
     }
 
     pub fn run(mut self) -> Result<(), SchedulerError> {
+        let mut min_prioritization_fees = u64::MAX;
+        let mut max_prioritization_fees = 0;
```

Comment:
sorry for being so picky, just put these on the `count_metrics` structure instead of doing local variables and passing them in.

Reply:
Yeah you are right, will do that.

Reply:
Done